wgpu-hal/src/dx12/suballocation.rs

```diff
@@ -183,6 +231,22 @@ pub(crate) fn free_texture_allocation(
     };
 }
 
+pub(crate) fn free_acceleration_structure_allocation(
+    device: &crate::dx12::Device,
+    allocation: AllocationWrapper,
+    allocator: &Mutex<GpuAllocatorWrapper>,
+) {
+    device
+        .counters
+        .acceleration_structure_memory
+        .sub(allocation.allocation.size() as isize);
+    match allocator.lock().allocator.free(allocation.allocation) {
+        Ok(_) => (),
+        // TODO: Don't panic here
```

Comment:
Is this todo for here or later

Reply:
This mimics the other destroys, that TODO was in the other two too so I duplicated it to this one too.